src/Mado/QueryBundle/Objects/RequestOptions.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Mado\QueryBundle\Objects;
+
+use Symfony\Component\HttpFoundation\Request;
+
+class RequestOptions
+{
+    private $requestOptions;
+
+    private function __construct(array $requestOptions)
+    {
+        $this->requestOptions = $requestOptions;
+    }
+
+    public static function fromRequest(Request $request, $entityAlias)
+    {
+        $requestOption = [];
+
+        $requestOption['filters']      = $request->query->get('filtering', []);
+        $requestOption['orFilters']    = $request->query->get('filtering_or', []);
+        $requestOption['sorting ']     = $request->query->get('sorting', []);
+        $requestOption['printing']     = $request->query->get('printing', []);
+        $requestOption['rel']          = $request->query->get('rel', '');
+        $requestOption['page']         = $request->query->get('page', '');
+        $requestOption['select']       = $request->query->get('select', $entityAlias);
+        $requestOption['filtering']    = $request->query->get('filtering', '');
+        $requestOption['limit']        = $request->query->get('limit', '');
+
+        return new self($requestOption);
+    }
+}
```

Comment:
Ok, but there are no getters. This isnt yet covered with test. New classes/objects should be.